Calculator/Calculator/Protocol/CalculateItem.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  CalculateItem.swift
+//  Calculator
+//
+//  Created by Daehoon Lee on 2023/05/31.
+//
+
+protocol CalculateItem { }
```

Comment:
[Declaring Protocol Adoption with an Extension](https://docs.swift.org/swift-book/documentation/the-swift-programming-language/protocols#Declaring-Protocol-Adoption-with-an-Extension) 문서를 보면 빈 구현의 경우 중괄호를 붙이는 것이 일반적일 것 같습니다.

```suggestion
protocol CalculateItem {}
```

Reply:
띄어쓰기 삭제하였습니다.😅

Reply:
그런데 공식문서에서도 `extension`에서 빈 구현으로 프로토콜 채택만 하는 예시의 경우 `{ }`로 적는 경우도 있더라고요. 한 프로젝트에서 일관된 컨벤션을 가져가는 게 제일 중요할 것 같습니다.